In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

In [2]:
%matplotlib inline

In [3]:
gdp_df = pd.read_csv('../data/gdp_percapita.csv')
gdp_df.head(6)

Country or Area  Year        Value  Value Footnotes
0     Afghanistan  2020  1970.560169              NaN
1     Afghanistan  2019  2065.036235              NaN
2     Afghanistan  2018  2033.804389              NaN
3     Afghanistan  2017  2058.400221              NaN
4     Afghanistan  2016  2057.067978              NaN
5     Afghanistan  2015  2068.265904              NaN

In [4]:
internet_df = pd.read_csv('../data/internet_use.csv', on_bad_lines='skip')
internet_df.head(6)


Country or Area  Year     Value  Value Footnotes
0     Afghanistan  2014  6.390000              NaN
1     Afghanistan  2013  5.900000              NaN
2     Afghanistan  2012  5.454545              NaN
3     Afghanistan  2011  5.000000              NaN
4     Afghanistan  2010  4.000000              NaN
5     Afghanistan  2009  3.550000              NaN

In [5]:
gdp_df.shape

(7176, 4)

There are 7176 rows and 4 columns in gdp dataframe

In [6]:
internet_df.shape

(4675, 4)

There are 4675 rows and 4 columns in internet_use dataframe

In [7]:
print(type(gdp_df['Country or Area']))
print(type(gdp_df.Year))
print(type(gdp_df.Value))
print(type(gdp_df['Value Footnotes']))

<class 'pandas.core.series.Series'>
<class 'pandas.core.series.Series'>
<class 'pandas.core.series.Series'>
<class 'pandas.core.series.Series'>


Show data types of columns:

In [8]:
gdp_df.columns

Index(['Country or Area', 'Year', 'Value', 'Value Footnotes'], dtype='object')

In [9]:
internet_df.columns


Index(['Country or Area', 'Year', 'Value', 'Value Footnotes'], dtype='object')

Drop Column 'Value Footnote'